### ISSUE TITLE
Can't resolve 'd3-scale

### ISSUE THREAD

**Issue by radjivC** [rma-consulting/react-easy-chart#108]
Hi, I have installed your package butI receive this error messages.
```
./node_modules/react-easy-chart/lib/shared.js
Module not found: Can't resolve 'd3-scale' in '/node_modules/react-easy-chart/lib'
``` 
any ideas?

Cheers

**Comment by njwest** [comment]
I am also having this issue, after `yarn install`

**Comment by njwest** [comment]
Was able to fix this problem by `yarn add`ing `d3-scale` and `react-faux-dom`

**Comment by radjivC** [comment]
work for me too thanks

**Comment by harunhasdal** [comment]
Fixed in 1.0.0
